playground/src/pages/index.vue

```diff
@@ -1,5 +1,84 @@
-<script setup lang="ts"></script>
+<script setup lang="ts">
+import {
+  basicRoutes,
+  perfRoutes,
+  eventsRoutes,
+  cameraRoutes,
+} from '../router/routes'
+
+const sections = [
+  { icon: '📦', title: 'Basic', routes: basicRoutes },
+  { icon: '🏎️', title: 'Perf', routes: perfRoutes },
+  { icon: '📣', title: 'Events', routes: eventsRoutes },
+  { icon: '📷', title: 'Camera', routes: cameraRoutes },
+]
+</script>
 
 <template>
-  <TheExperience />
+  <div
+    class="
+  container mx-auto max-w-3xl
+  font-sans text-xs color-gray
+  bg-white 
+  "
+  >
+    <div class="mx-4">
+      <div
+        class="
+    mt-24 mb-12 text-center align-baseline items-center gap-6
+    sm:mt-16 sm:mb-6 sm:text-left sm:flex sm:flex-row-reverse
+    "
+      >
+        <div>
+          <img
+            src="/logo.svg"
+            alt="Cientos logo"
+            class="max-w-36 sm:max-w-xs align-baseline"
+          >
+        </div>
+        <div class="sm:w-2/3">
+          <h1
+            class="
+        w-auto max-w-75 mx-auto text-5xl text-zinc-700 mb-3
+        sm:mx-none sm:w-1/2 sm:max-w-72
+        "
+          >
+            <span class="text-cientos-blue">TresJS</span> Playground
```

Comment:
Add named colors to the UnoCSS config in /playground/vite.config.js

```
     UnoCSS({
       theme: {
         colors: {
           'tresjs-blue': '#82dbc5',
           'tresjs-orange': '#fbb03b',
         },
       },
```

Use that here:

```
            <span class="text-tresjs-orange">TresJS</span> Playground
```